Algorithms.Tests/Search/KmpSearcherTests.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Algorithms.Search;
+using NUnit.Framework;
+using NUnit.Framework.Internal;
+
+using Utilities.Exceptions;
+using Utilities.Extensions;
+
+namespace Algorithms.Tests.Search
+{
+    public class KmpSearcherTests
```

Comment:
Codacy found an issue: [Add a 'protected' constructor or the 'static' keyword to the class declaration.](https://app.codacy.com/manual/siriak/C-Sharp/pullRequest?prid=6285773)